src/generators/minecraft-mutant-character/MutantCharacter.res

```diff
@@ -0,0 +1,507 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "mutantgenerator"
+
+let name = "Mutant Character"
```

Comment:
Let's rename this generator:

Directory name: `minecraft-mutant-character`

File name: `MinecraftMutantCharacterGenerator.res`

id: `minecraft-mutant-character`

name: `Mutant Character`

Reply:
Done